test/e2e-tests.sh

```diff
@@ -35,4 +35,19 @@ header "Running Go e2e tests"
 go_test_e2e ./test || failed=1
 
 (( failed )) && fail_test
+
```

Comment:
`install_pipeline_crd` needs to be uncommented above :angel:

Reply:
We have planned to run cli command without installing the pipeline where all the commands should fail. Further, we will run install_pipeline_crd and create task, taskrun, pipeline, and pipelinerun then run the cli commands which should execute without any error.

Reply:
ah ok :+1: